stores/governance/src/main/resources/db/store/postgresql/V0_1100_1__init.sql

```diff
@@ -41,6 +41,64 @@ create table voting_procedure
     primary key (tx_hash, index)
 );
 
+CREATE TABLE committee_registration
+(
+    tx_hash         varchar(64) not null,
+    cert_index      int         NOT NULL,
+    cold_key        varchar,
+    hot_key         varchar,
+    slot            bigint,
+    block           bigint,
+    block_time      bigint,
+    update_datetime timestamp,
+    PRIMARY KEY (tx_hash, cert_index)
+);
+
+CREATE TABLE committee_de_registration
+(
+    tx_hash         varchar(64) not null,
+    cert_index      int         NOT NULL,
+    anchor_url      varchar(256),
+    anchor_hash     varchar(64),
+    cold_key        varchar     NOT NULL,
+    slot            bigint,
+    block           bigint,
+    block_time      bigint,
+    update_datetime timestamp,
+    PRIMARY KEY (tx_hash, cert_index)
+);
+
+CREATE TABLE delegation_vote
```

Comment:
We may need additional index for find delegation for a stake address etc. But let's do it in another PR.